src/components/Select/index.tsx

```diff
@@ -28,9 +40,10 @@ const StyledGrommetSelect = styled(
 `;
 
 // Set button to occupy parent's width, and a default width.
-const Container = styled.div`
+const Container = styled(withTheme(Flex))`
```

Comment:
`styled` already has access to the theme, no need for `withTheme`. Also, why do we need a `Flex` here, the div is just fine (and I see that it affected the height of the Select in the screenshot)